workflow/controller/controller_test.go

```diff
@@ -308,3 +312,43 @@ func TestAddingWorkflowDefaultComplexTwo(t *testing.T) {
 	assert.Contains(t, workflow.Labels, "label")
 	assert.Contains(t, workflow.Annotations, "annotation")
 }
+
+func TestNamespacedController(t *testing.T) {
+	kubeClient := fake.Clientset{}
+
+	kubeClient.AddReactor("create", "selfsubjectaccessreviews", func(action k8stesting.Action) (handled bool, ret runtime.Object, err error) {
+		selfSubjectAccessReview := reflect.ValueOf(action).FieldByName("Object").Elem().Elem().Field(2).Field(0).Elem()
```

Comment:
should not need to use reflection, action.(k8stesting.CreateAction)

Reply:
https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/statefulset/stateful_pod_control_test.go